order.php

```diff
@@ -33,67 +25,52 @@
+                </div>
               </div>
               <!-- BTC Amount -->
               <div class="bnomics-amount">
               <div class="bnomics-bg">
                 <!-- Order Status -->
                 <div class="bnomics-order-status-wrapper">
-                  <span class="bnomics-order-status-title" ng-show="order.status == -1" ng-cloak ><?=__('To confirm your order, please send the exact amount of <strong>BTC</strong> to the given address', 'edd-blockonomics')?></span>
+                  <span class="bnomics-order-status-title" ng-show="order.status == -1" ng-cloak ><?=__('To pay, send exactly this BTC {{altcoinselect}} amount')?></span>
                   <span class="warning bnomics-status-warning" ng-show="order.status == -3" ng-cloak><?=__('Payment Expired (Use the browser back button and try again)', 'edd-blockonomics')?></span>
                   <span class="warning bnomics-status-warning" ng-show="order.status == -2" ng-cloak><?=__('Payment Error', 'edd-blockonomics')?></span>
                 </div>
-                    <h4 class="bnomics-amount-title" for="invoice-amount" ng-hide="order.status == -3">
-                     {{order.satoshi/1.0e8}} BTC
-                    </h4>
-                    <div class="bnomics-amount-wrapper" ng-hide="order.status == -3">
-                      <hr class="bnomics-amount-seperator"> ≈
-                      <span ng-cloak>{{order.value}}</span>
-                      <small ng-cloak>{{order.currency}}</small>
+                    <div>
+                    <ul ng-click="blockonomics_amount_click()" id="bnomics-amount-input" class="bnomics-amount-input">
+                        <li id="bnomics-amount-copy">{{order.satoshi/1.0e8}} BTC</li>
+                        <li>{{crypto.code | uppercase}}</li>
+                        <li class="bnomics-grey"> ≈ </li>
+                        <li class="bnomics-grey">{{order.value}}</li>
+                        <li class="bnomics-grey">{{order.currency}}</li>
+                    </ul>
                     </div>
               <!-- Bitcoin Address -->
-                <div class="bnomics-address" ng-hide="order.status == -3">
-                  <input ng-click="btc_address_click()" id="bnomics-address-input" class="bnomics-address-input" type="text" ng-value="order.address" readonly="readonly">
-                  <i ng-click="btc_address_click()" class="material-icons bnomics-copy-icon">file_copy</i>
-                </div>
-                <div class="bnomics-copy-text" ng-hide="order.status == -3 || copyshow == false" ng-cloak>Copied to clipboard</div>
+              <div class="bnomics-address">
+                    <div class="bnomics-address-text" ng-hide="address_copyshow" ng-cloak><?=__('To this bitcoin{{crypto.name | lowercase}} address', 'edd-blockonomics')?></div>
+                    <div class="bnomics-copy-address-text" ng-show="address_copyshow" ng-cloak><?=__('Copied to clipboard', 'edd-blockonomics')?></div>
```

Comment:
The copied to clipboard text does not display onclick. This js can be copied from the woocommerce plugin:
https://github.com/blockonomics/woocommerce-plugin/blob/master/js/app.js#L104-L156

Reply:
Clicking on the bitcoin amount should copy the bitcoin amount to the clipboard. You can complete this using  https://github.com/blockonomics/woocommerce-plugin/blob/master/js/app.js#L149-L152 and https://github.com/blockonomics/woocommerce-plugin/blob/master/templates/blockonomics_checkout.php#L74

Reply:
Done